padle/padle.py

```diff
@@ -0,0 +1,801 @@
+                    # need to send message if a user is mid-game
+                    if await self.config.user(user).start() and not await self.config.user(user).done():
+                        await user.send("The PADle expired; a new one is available.")
+                    await self.config.user(user).start.set(False)
+                    await self.config.user(user).done.set(False)
+                    await self.config.user(user).score.set([])
+                    await self.config.user(user).edit_id.set("")
+                    await self.config.user(user).channel_id.set("")
+                subbed_users = await self.config.subs()
+                for userid in subbed_users:
+                    user = self.bot.get_user(userid)
+                    if user is not None:
+                        await user.send("PADle #{} is now available!".format(await self.config.num_days()))
+            except asyncio.CancelledError:
+                raise
+            except Exception:
+                logger.exception("Error in loop:")
+
+    @padleadmin.command()
+    async def fullreset(self, ctx):
+        """Resets all stats and information"""
+        confirmation = await get_user_confirmation(ctx,
+                                                   "Fully reset all stats and data in PADle? You cannot undo this!")
+        if confirmation is None:
+            return await send_cancellation_message(ctx, "Confirmation timeout.")
+        if not confirmation:
+            return await send_cancellation_message(ctx, "Nothing was reset.")
+
+        await self.config.padle_today.set(3260)
+
+        await self.config.tmrw_padle.set(0)
+        await self.config.num_days.set(1)
+        await self.config.subs.set([])
+        await self.config.all_scores.set([])
+        await self.config.save_daily_scores.set([])
+        all_users = await self.config.all_users()
+        for userid in all_users:
+            user = self.bot.get_user(userid)
+            if user is None:
+                continue
+            await self.config.user(user).todays_guesses.set([])
+            # need to send message if a user is mid-game
+            if await self.config.user(user).start() and not await self.config.user(user).done():
+                try:
+                    await user.send("A full reset occured, the PADle expired.")
+                except:
+                    pass
+            await self.config.user(user).start.set(False)
+            await self.config.user(user).done.set(False)
+            await self.config.user(user).score.set([])
+            await self.config.user(user).edit_id.set("")
+            await self.config.user(user).channel_id.set("")
+            await self.config.user(user).all_guesses.set({})
+        await ctx.tick()
+
+    @padleadmin.command()
+    async def add(self, ctx, *, list):
+        """Adds CSV of monster IDs to the list of possible PADles"""
+        clean_list = re.sub(" ", "", list).split(",")
```

Comment:
You can use `add(self, ctx, *ids: int)` to have ids automatically be a list of integers.  It wouldn't allow commas anymore though, so you'd have to change the docstring